packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -136,7 +141,7 @@ class DateTimePicker extends React.Component {
     // It is used for date/time manipulation and used to calculate the missing/ambiguous hour.
     // The dateValue and timeValue are tracked outside of the react state to limit the number of renderings that occur.
     this.dateValue = DateUtil.formatMomentDate(this.state.dateTime, this.state.dateFormat) || '';
-    this.timeValue = DateTimeUtils.hasTime(this.props.value) ? DateUtil.formatISODate(this.props.value, 'HH:mm') : '';
```

Comment:
Is formatISODate still a used helper?

Reply:
`formatISODate` is still used to format dates but is not used to format times anymore.